pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go

```diff
@@ -22,8 +22,8 @@ type Multus struct{}
 type LinuxBridge struct{}
 
 type KubeMacPool struct {
-	StartPoolRange string `json:"startPoolRange,omitempty"`
-	EndPoolRange   string `json:"endPoolRange,omitempty"`
+	RangeStart string `json:"RangeStart,omitempty"`
```

Comment:
RangeStart -> rangeStart

Reply:
Only the one on right. Left-hand-side RangeStart should remain this way